src/createAnimatedComponent.tsx

```diff
@@ -602,14 +605,18 @@ export default function createAnimatedComponent(
       this._attachInlineProps();
     }
 
-    _setComponentRef = setAndForwardRef<Component>({
+    _setComponentRef = setAndForwardRef<Component | HTMLElement>({
       getForwardedRef: () =>
         this.props.forwardedRef as MutableRefObject<
           Component<Record<string, unknown>, Record<string, unknown>, unknown>
         >,
       setLocalRef: (ref) => {
         // TODO update config
-        const tag = findNodeHandle(ref);
+
+        const tag = isWeb()
```

Comment:
Let's add `const IS_WEB = isWeb();` somewhere at the top of the file and use memoized value here:
```suggestion
        const tag = IS_WEB
```